openapi/components/schemas/plaid/PlaidCallbackResponse.yaml

```diff
@@ -0,0 +1,19 @@
+type: object
+required:
+  - status
+  - message
+properties:
+  status:
+    type: string
+    enum:
+      - PROCESSING
```

Comment:
Do we need more statuses here? Maybe at least ACTIVE and FAILED? Alternatively, if it's always going to be processing in this response, maybe we don't need it?